daprdocs/content/en/getting-started/quickstarts/pubsub-quickstart.md

```diff
@@ -423,7 +423,7 @@ dotnet build
 Run the `checkout` publisher service alongside a Dapr sidecar.
 
 ```bash
-dapr run --app-id checkout --resources-path ../../../components -- dotnet run
+dapr run --app-id checkout --app-port 7006 --resources-path ../../../components -- dotnet run
```

Comment:
I like the idea here, just two things:
1) this change should be applied to the dapr run on `order-processor` - since that is the receiver of the POST/message.  Is that what you meant to do?
2) we should use port 7001 and match the underlying quickstart readme exactly.

The source of truth is here.  @msfussell and @hhunter-ms note the docs have drifted a bit from readmes and I suggest we coordinate an update. 
  https://github.com/dapr/quickstarts/tree/master/service_invocation/csharp/http